lib/API/JSON/v1/MP2Migrator.php

```diff
@@ -16,6 +16,8 @@ public function __construct() {
     $this->MP2Migrator = new \MailPoet\Config\MP2Migrator();
   }
 
+  private $MP2Migrator;
+
```

Comment:
It's just a small coding style thing but I think properties should better be defined before contructor?

Reply:
You're right, good catch. Will fix it.